chroma-manager/tests/feature/cli/features/steps/standard_cli.py

```diff
@@ -1,8 +1,3 @@
-#
-# ========================================================
-# Copyright (c) 2012 Whamcloud, Inc.  All rights reserved.
-# ========================================================
-
```

Comment:
it is test code and files in the same directory (as with most test files) do not have Copyright headers. if we want to change this we should do it in a separate patch